### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by stevepeak**
- faster report processing
- removed ignore because no longer processing these paths

Thank you! 👍

### CODE REVIEW

**Comment by Lucashuang0802** [comment]
https://codecov.io/gh/Lucashuang0802/LHToolbar/tree/77e774e59a6b6273ec18738de45b00d1de91a222
I think I only want to track this folder: `LHToolbar`. @stevepeak

File: codecov.yml [hunk 1]
```diff
@@ -2,12 +2,7 @@ coverage:
   precision: 2
   round: nearest
   range: "60...100"
-  ignore:
-  - LHToolbarDemoCocoaPodsTest/*
-  - LHToolbarTests/*
-  - LHToolbarDemo/*
-  - Pods/*
```
Comment (Lucashuang0802) [inline codecov.yml:5]: That's exactly what I wanted to ignore.

File: .travis.yml [hunk 2]
```diff
@@ -9,4 +9,4 @@ script:
   - pod lib lint --allow-warnings
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash);
+  - bash <(curl -s https://codecov.io/bash) -J LHToolbar
```
Comment (Lucashuang0802) [inline .travis.yml:12]: any differences to add this?

Comment (stevepeak) [inline .travis.yml:12]: Speeds up processing by only running coverage for the "LHToolbar" project

File: codecov.yml (hunk 1 above)
Comment (stevepeak) [inline codecov.yml:5]: These will be ignored because of the `-J` below. Codecov will only process that project and skip the ones above.

**Pull request closed without merging**